tests/bsim/bluetooth/hci_uart/compile.sh

```diff
@@ -12,6 +12,8 @@ source ${ZEPHYR_BASE}/tests/bsim/compile.source
 
 app=tests/bsim/bluetooth/ll/conn conf_file=prj_split_hci_uart.conf \
     cmake_extra_args=-DEXTRA_DTC_OVERLAY_FILE=hci-uart.overlay compile
+app=tests/bsim/bluetooth/ll/conn conf_file=prj_split_hci_uart.conf conf_overlay=psa_overlay.conf \
```

Comment:
Is there any reason to do this in the set of tests meant to test the HCI UART driver? (controller and host in separate nrf52 devices connected over UART)
Or can this be done in the plain ll/conn tests?

Reply:
That's the only test that I found in `bsim/bluetooth` that had `CONFIG_BT_HOST_CRYPTO` enabled and this is what I needed to test changes in `subsys/bluetooth/host/crypto_psa.c`